### DISCUSSION TITLE
Cloud PC

### DISCUSSION THREAD

**Discussion by totalitarian** [alvr-org/ALVR#377] [General]
Hi, which version supports running ALVR on a cloud PC? I guess version 13 doesn't

```
Starting service: Intent { cmp=com.polygraphene.alvr/.ChangeSettings (has extras) }
Error: Not found; no service started.
```

**Comment by zmerp** [comment]
Cloud PC will be supported soon, sometimes after the v14 release. In the meantime you can use ALVR ev11 which is the last version that supports ChangeSettings.

**Reply by totalitarian** [reply]
Thanks!

**Comment by JosNun** [comment]
As of v14, ALVR supports streaming from a cloud PC. There's instructions on how to set it up [in the wiki](https://github.com/alvr-org/ALVR/wiki/ALVR-client-and-server-on-separate-networks).
